scss/core/variables/_colors.scss

```diff
@@ -0,0 +1,122 @@
+
+// Color system
+
+$white:    #fff !default;
+$gray-100: #F9F9F9 !default;
+$gray-200: #b9babe !default;
+$gray-300: #A2A4A9 !default;
+// $gray-400: #A2A2A2 !default; // TODO: do we need this gray?
```

Comment:
Just checking if this is left in on purpose.

Reply:
I did leave them there on purpose, but I suppose easy enough to add back in if needed. Will delete.